microsoft-edge/devtools-guide-chromium/whats-new/2023/05/devtools-113.md

```diff
@@ -0,0 +1,164 @@
+ms.date: 05/24/2023
+---
+# What's New in DevTools (Microsoft Edge 113)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+> [!TIP]
+> The **Microsoft Build 2023** conference was on May 23-25, 2023.  Learn more about new features for memory, performance, and production debugging in DevTools and new capabilities for PWAs in the sidebar, WebView2, and Chat Plugins in the following videos:
+> * [Microsoft Edge | State of web developer tools](https://www.youtube.com/watch?v=yDFmQNu3TSg&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+> * [Microsoft Edge | Building Progressive Web Apps for the sidebar](https://www.youtube.com/watch?v=9u8lRzRUayw&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+> * [Microsoft Edge | Bringing WebView2 to Microsoft Teams and beyond](https://www.youtube.com/watch?v=s3tDUvaoCP4&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+> * [Building Chat Plugins for Microsoft Bing and Edge](https://www.youtube.com/watch?v=Q-5M7EYjl6U&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+
+
+<!-- ====================================================================== -->
+## Debug JavaScript error stack traces with the experimental Crash Analyzer tool
+
+<!-- Subtitle: Turn on the "Enable Crash Analyzer" experiment and paste in your error stack trace with source map references for faster debugging. -->
+
+The new **Crash analyzer** tool is available as an experiment in Microsoft Edge 113.  You can access the **Crash analyzer** tool by selecting the **Enable Crash Analyzer** experiment in **Settings** > **Experiments**.  See [Turning an experiment on or off](../../../experimental-features/index.md#turning-an-experiment-on-or-off).
+
+With the **Enable Crash Analyzer** experiment turned on, you can now access the **Crash analyzer** tool.  In the **Crash analyzer** tool, you can input a JavaScript stack trace, such as for non-fatal JavaScript exceptions, and then have your sourcemaps applied to the stack trace so that you can debug faster.
+
+You can report non-fatal JavaScript exceptions or similar data to tools such as Azure Application Insights:
+
+1. You can copy the stack trace from one of these exceptions out of Application Insights, and then paste the stack trace into the left pane of the **Crash analyzer** tool.
+
+1. You will also need to include a custom section called `Source modules`, which contains the sourcemap references to the JavaScript functions that were part of the error stack trace.
+
+1. To produce the stack traces that are needed by the **Crash analyzer** tool, you can use the [Crash Analyzer Support](https://www.npmjs.com/package/@microsoft/edge-devtools-crash-analyzer-support) npm package.
+
+1. In the **Crash analyzer** tool, click the **Analyze** button.  The right-hand pane shows the original file and function names that make up the stack trace:
+
+   ![Using the Crash analyzer tool to debug a non-fatal JavaScript exception](./devtools-113-images/crash-analyzer-tool.png)
```

Comment:
move png to below the next step?

Reply:
I kind of like where it is. We can draw red boxes around the Analyze button and the right-hand pane

Reply:
has unminified & red boxes